schedule/yam/agama_ext4_xfs_unattended.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: agama unattended xfs
```

Comment:
@jknphy any suggestions about the file name? The scheduled modules for ext4 and xfs are the same. So I named the yaml schedule file as  agama_ext4_xfs_unattended.yaml, will it make users feel confuse if I change as below:
```
name: agama unattended xfs or ext4
description: >
  Perform agama unattended installation with xfs or ext4.
```
or any other suggestions here?

Reply:
agama_non_default_root_filesystem.yaml